### ISSUE TITLE
styles not found

### ISSUE THREAD

**Issue by morenoh149** [keystonejs/generator-keystone#168]
I get `Failed to load resource: the server responded with a status of 404 (Not Found) http://localhost:3000/styles/site.css` on a brand new site with Swig and stylus templates.
![screen shot 2016-04-27 at 1 58 56 am](https://cloud.githubusercontent.com/assets/587438/14843148/a22d8348-0c1b-11e6-9a68-4260bded06dd.png)

**Comment by mxstbr** [comment]
No, you need to change the `.css` once so the compiler runs, had the same problem for `foundation.nodejs.org`. See the fix here: https://github.com/nodejs/foundation.nodejs.org/pull/2

**Comment by morenoh149** [comment]
:/ then the generator should display such instructions, or ship with a precompiled .css file. I'll look into it.
